mmengine/hooks/ema_hook.py

```diff
@@ -172,6 +171,7 @@ def after_load_checkpoint(self, runner, checkpoint: dict) -> None:
         Args:
             runner (Runner): The runner of the testing process.
         """
+        from mmengine.runner.checkpoint import load_state_dict
         if 'ema_state_dict' in checkpoint and runner._resume:
```

Comment:
can also add a `resume` property to runner

Reply:
`Runner` has defined the `resume` method:

![image](https://user-images.githubusercontent.com/57566630/189336509-4be33233-ce39-4a7c-8b1a-db25fcf09c57.png)